custom/conf/app.ini.sample

```diff
@@ -242,6 +242,11 @@ SQLITE_TIMEOUT = 500
 ITERATE_BUFFER_SIZE = 50
 ; Show the database generated SQL
 LOG_SQL = true
+; Maximum number of DB Connect retries
+DB_RETRIES = 10
+; Backoff time per DB retry (seconds)
+DB_RETRY_BACKOFF = 3
```

Comment:
Tiny nit: As this value is `time.Duration`, it should probably be set to `3s` instead of just `3`

Reply:
Its all good.  One guy won't catch everything!  But hang on a sec...  So, when I made these insertions, I wanted to follow the existing patterns.  I looked over the existing configs in the ini & unfortunately, its actually a mixed bag throughout the ini file.  You have many settings as constant and many as Duration.  So what I did was I went with the existing pattern in just the [database] area which there is only one other one & its a constant:

```
SQLITE_TIMEOUT = 500
```

FWIW, I would normally recommend config settings in constants so as not to burden users with implementation or notation details.  But more importantly, it should be consistent one way or the other.  Having said that, I don't mind making this change to use the golang specific construct if that is really what you want; I'd also need to tweak the code to adapt to that. Let me know.  no biggie really either way.

Reply:
Yes, sadly some legacy settings are constants, and it would be difficult to change those old ones without breaking some installs, and so when adding new things we try to have them be duration.

Thanks for all your hard work 😄